ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -1,10 +1,33 @@
 #include "../PrjFSKext/kernel-header-wrappers/vnode.h"
 #import <XCTest/XCTest.h>
 #include "../PrjFSKext/KauthHandlerTestable.hpp"
+#include "../PrjFSKext/PerformanceTracing.hpp"
 #include "MockVnodeAndMount.hpp"
 
 using std::shared_ptr;
 
+#define KAUTH_RESULT_ALLOW    (1)
+#define KAUTH_RESULT_DENY     (2)
+#define KAUTH_RESULT_DEFER    (3)
+
+proc_t vfs_context_proc(vfs_context_t ctx)
+{
+    return NULL;
+}
+
+extern "C" int proc_pid(proc_t);
+int proc_pid(proc_t)
+{
+    return 1;
+}
+
+extern "C" void proc_name(int pid, char * buf, int size);
+char * procname;
+void proc_name(int pid, char * buf, int size)
+{
+    buf = procname;
```

Comment:
This essentially a no-op, `buf` is passed by value. You'll need to use `strlcpy()` to write to the supplied buffer. (But then `procname` is never initialised; is this function called by the current tests? If not, perhaps just put an `assert(false)` in here until we actually need to implement it properly.)